deepfence_server/reporters/scan/scan_reporters.go

```diff
@@ -697,7 +697,8 @@ func GetScanResultDiff[T any](ctx context.Context, scan_type utils.Neo4jScanType
 	MATCH (n:` + string(scan_type) + `{node_id: $base_scan_id}) -[r:DETECTED]-> (d)
 	WHERE NOT EXISTS {MATCH (m:` + string(scan_type) + `{node_id: $compare_to_scan_id}) -[:DETECTED]-> (d)}
 	OPTIONAL MATCH (d) -[:IS]-> (e)
-	WITH apoc.map.merge( e{.*}, d{.*, masked: coalesce(d.masked or r.masked, false), name: coalesce(e.name, d.name, '')}) AS d` +
+	WITH apoc.map.merge( e{.*}, d{.*, masked: coalesce(d.masked or r.masked or e.masked, false), 
```

Comment:
IIRC we said `r.masked` was not needed anymore - is it?

Reply:
This is required for the Scan level masking.